homeassistant/components/tplink_omada/config_flow.py

```diff
@@ -0,0 +1,126 @@
+"""Config flow for TP-Link Omada integration."""
+from __future__ import annotations
+
+import logging
+from types import MappingProxyType
+from typing import Any
+
+from tplink_omada_client.exceptions import (
+    ConnectionFailed,
+    LoginFailed,
+    OmadaClientException,
+    UnsupportedControllerVersion,
+)
+from tplink_omada_client.omadaclient import OmadaClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME, CONF_VERIFY_SSL
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_VERIFY_SSL, default=True): bool,
+        vol.Required("site", default="Default"): str,
```

Comment:
Could we move this to a 2nd step and list the actual available sites + skip the step if the user has only 1 site? (probably most people)

Reply:
Done. I might need to revisit this to support multiple sites, but as you say, 99% of people (including me!) will only have one site.